microprofile/grpc/core/src/main/java/io/helidon/microprofile/grpc/core/ModelHelper.java

```diff
@@ -90,56 +89,45 @@
     }
 
     /**
-     * Get privileged action to obtain methods declared on given class.
-     * If run using security manager, the returned privileged action
-     * must be invoked within a doPrivileged block.
+     * Get collection of methods declared on given class.
      *
      * @param clazz class for which to get the declared methods.
-     * @return privileged action to obtain methods declared on the {@code clazz} class.
-     * @see java.security.AccessController#doPrivileged(java.security.PrivilegedAction)
+     * @return methods declared on the {@code clazz} class.
      */
-    public static PrivilegedAction<Collection<? extends Method>> getDeclaredMethodsPA(final Class<?> clazz) {
-        return () -> Arrays.asList(clazz.getDeclaredMethods());
+    public static Collection<? extends Method> getDeclaredMethods(final Class<?> clazz) {
```

Comment:
This seems to be a public class and public method. I think it was intended for our use, but not sure. 
As this is a backward incompatible change - we should either document (and explain) then change as an intended break that must be fixed between minor version, or create separate methods without `PrivilegedAction` and mark these as deprecated, or mark these methods as deprecated and change the signature in 3.0.0

Reply:
I think they are just public for use outside of `core` in other parts of the GRPC implementation. My preference would be to document and change rather than go through the whole deprecation dance for no practical reason.